docs/ad/api.md

```diff
@@ -151,6 +138,133 @@ Options | Description | Type | Required
 }
 ```
 
+To set a category field for high cardinality:
+
+#### Request
+
+```json
+POST _opendistro/_anomaly_detection/detectors
+{
+  "name": "Second Test Detector 2",
+  "description": "ok rate",
+  "time_field": "@timestamp",
+  "indices": [
+    "host-cloudwatch"
+  ],
+  "category_field": [
+    "host"
+  ],
+  "feature_attributes": [
+    {
+      "feature_id": "latency_max",
```

Comment:
No need to provide "feature_id" when create detector. AD backend will fill this field with UUID automatically.